index.bs

```diff
@@ -237,6 +227,39 @@ The <dfn>AV1MultiFrameSampleGroupEntry</dfn> documents samples that contain mult
 class AV1MultiFrameSampleGroupEntry extends VisualSampleGroupEntry('av1m') {
 }
 ```
+AV1 Forward Access sample group entry {#forwardaccesssamplegroupentry}
+----------------------------------------------------------------
+
+### Definition ### {#forwardaccesssamplegroupentry-definition}
+
+<pre class='elementdef'>
+	Group Type: av1f
+	Container: Sample Group Description Box ('sgpd')
+	Mandatory: No
+	Quantity: Zero or one.
+</pre>
+
+
+### Description ### {#forwardaccesssamplegroupentry-description}
+
+The <dfn>AV1ForwardAccessSampleGroupEntry</dfn> documents samples that contain multiple frames.
+
+
+### Syntax ### {#forwardaccesssamplegroupentry-syntax}
+
+```
+class AV1ForwardAccessSampleGroupEntry extends VisualSampleGroupEntry('av1f') {
+  unsigned int(1) initial_display;
+  unsigned int(7) fwd_distance;
+}
+```
+
+### Semantics ### {#forwardaccesssamplegroupentry-semantics}
+
+The <dfn>initial_display</dfn> fields indicates if the processing of this sample will produce an output frame or not.
+
+The <dfn>fwd_distance</dfn> field indicates the number of samples whose decoding can be skipped before encountering the sample outputting the next frame. 0 means the next sample.
```

Comment:
Will muxers have to look at frame headers to determine this value?

Reply:
Indeed. Once a muxer parses a Key Frame with show_frame=0, it would have to look forward in the stream to see if there is a show_existing_frame pointing to it, to be able to set this distance. The idea is that the demuxer would not have to do this job.

Reply:
Generating this sample group is indeed complex, but it is optional so that is less problematic. To generate it, I think either you have a good integration with your encoder and you will be able to generate it easily or you would have to do complex parsing and reference buffer management. Maybe the frame ids can help, but you can also do without by tracking buffer references.